fastlane/lib/fastlane/actions/automatic_code_signing.rb

```diff
@@ -153,7 +190,7 @@ def self.return_value
       end
 
       def self.authors
-        ["mathiasAichinger", "hjanuschka"]
+        ["mathiasAichinger", "hjanuschka", "mindera.com", "p4checo", "portellaa", "aeons"]
```

Comment:
Are those the GitHub/Twitter usernames? I can't find mindera.com

Reply:
`mindera.com` indeed isn't 😅 

We added it initially on cosigner as a hat tip to our company, and I copied it here. 😛 
Since only usernames are allowed/expected, I'll remove it